src/components/customerPortal/PortalInvoiceListItem.tsx

```diff
@@ -49,6 +49,7 @@ interface PortalInvoiceListItemProps {
   invoice: PortalInvoiceListItemFragment
   translate: Function
   className?: string
+  documentLocale: string
```

Comment:
Should be of type LocaleEnum

Reply:
if it complain about the 'en' being a non valid value, we need to update the LocaleEnum definition to be an object `as const`

Let me know if that's happening

Reply:
changed